packages/platforms/browser/lib/delivery.ts

```diff
@@ -43,13 +43,16 @@ function createBrowserDeliveryFactory (
   return function browserDeliveryFactory (endpoint: string): Delivery {
     return {
       async send (payload: TracePayload) {
+        const body = JSON.stringify(payload.body)
+
         payload.headers['Bugsnag-Sent-At'] = clock.date().toISOString()
+        payload.headers['Bugsnag-Uncompressed-Content-Length'] = body.length.toString()
```

Comment:
This doesn't seem necessary because we don't compress the payload so would just be a repeat of `Content-Length`

It's not an accurate size though because:

- browsers can and will mutate the body before starting the request, e.g. Safari applies unicode normalisation so the body here isn't necessarily the same as the body that gets sent in the request
- `fetch` encodes strings as UTF-8 but JS strings are UTF-16
- `String#length` returns the number of UTF-16 code units in the string, not the number of bytes

I think it would be better not to add this header as the backend can use `Content-Length` for the same purpose and that is calculated by the browser based on the actual request body